File: hermetic-cc-toolchain-with-clang/main/hello-world.cc
#include <iostream>

std::string get_greet(const std::string& who) {
  return "Hello " + who;
}


int main(int argc, char** argv) {
  std::string who = "world";
  std::cout << get_greet(who) << std::endl;
  return 0;
}

File: python-tutorial/stage1/main/hello-world.py
import sys
from datetime import datetime


def get_greet(who: str) -> str:
    return f"Hello {who}"


def print_localtime() -> None:
    print(datetime.now())


def main() -> None:
    who = "world"

    if len(sys.argv) > 1:
        who = sys.argv[1]

    print(get_greet(who))
    print_localtime()


if __name__ == "__main__":
    main()
